app/jobs/clock/activate_subscriptions_job.rb

```diff
@@ -2,10 +2,16 @@
 
 module Clock
   class ActivateSubscriptionsJob < ApplicationJob
+    include Sentry::Cron::MonitorCheckIns
+
     queue_as 'clock'
 
     unique :until_executed, on_conflict: :log
 
+    if ENV['SENTRY_ENABLE_CRONS']
```

Comment:
it would be awesome to create a concern for it that take the slug in params, wdyt?